internal/cmd/plugin/status/cmd.go

```diff
@@ -41,6 +41,10 @@ func NewCmd() *cobra.Command {
 			ctx := cmd.Context()
 			clusterName := args[0]
 
+			if err := plugin.ConfigureColor(cmd); err != nil {
+				return err
+			}
+
```

Comment:
I think we could actually do this at `PersistentRunE` level, so this will work in every `kubectl cnpg` subcommand.

Reply:
Take into consideration that the color flag is only applicable to the hibernate status and status commands. Therefore, I have not designated it as a global flag, but rather assigned it at the command level.

Reply:
Yes, but everything will be ready if we implement another command requiring it.
With your implementation, we need to remember to set the colorizer in the new command, too.